### ISSUE TITLE
Any help would be greatly appreciated

### ISSUE THREAD

**Issue by taigalover** [erengy/taiga#469]
i have been using tyga mainly for its unbelievably smooth way to sync my mal account to my desktop. but people reccomend using its rss capabilites. so i tried but i simply managed to confuse myself more. so please give me a very brief introduction to what an rss is because the settings menu has 2 different options and they both need something different and i just need something which helps me search up and find a bunch of different animes. any help greatly appreaciated

**Comment by Thomasedv** [comment]
Some simple explanation for me which got little clue, but worked my way around with it. 

RSS is kinda like a feed or list, with links to your torrentfiles/episodes. One feed is the one to the most recent releases, this is what Taiga uses in it's automatic checking that goes on as often as you'd like. When it sees an episode that meets it's criteria (e.g. it's airing, an episode you haven't seen and not downloaded already) it marks that episode. (And downloads the torrent file it you want right there. I don't for a few reasons but it's an option. )

The other RSS feed is the search option, it's what you get when you search for a series in the top left corner in the torrent tab. Depending on the feed criteria, you can get only english translated shows, 1080p etc. It all depends on a sites specific RSS and options that site you want to use provides. 

I use nyaa.si for example. The first link is the same results that you'd get if you search for english translated shows on that site, and sort by new. The other link is a link which searches for a english translated series with a name that matches the provided query(name).

**Comment by taigalover** [comment]
correct me if im wrong this is what i understood:

1.  For the first rss field, this is the field which allow me to search for updates for a specific show like, for 
    example "one piece".if i put an rss for one piece it will then notify me a new episode every week and 
    also auto download (depending on the settings).so the way i get this rss for one piece would be, to go  
    to nyaa.si and type one piece and choose english-translated, one the page loads i click on "rss" on the 
    top bar and and copy the url of the loaded page, then i paste this in the first rss field in taiga.this is 
    what i get "https://nyaa.si/?page=rss&q=one+piece&c=1_2&f=0". but this only shows the first page 
    of everything in the taiga query though, is this normal?

2. The second rss feed is to give taiga a refrence to search for your query. like if i put an rss from nyaa.si 
    and searched something like "one piece" it would return me results as if i was searching from nyaa.si
    itself. though i have tried to get an rss like this to no avail, in which i would appreciate some help.

**Comment by taigalover** [comment]
ahhh.

i finally got what you said at the end about adding in %title%. i treid it and it worked like a charm.

so i finally know how to use the second rss field. so i have no questions about the first field.
but how about the first, particularly if i want to keep up with many different shows, how do i use many rss at the same time? and how do i apply my prefrences into them? ive checked other posts about this and i got confused.

**Comment by Thomasedv** [comment]
The first field isn't show specific. You should only give it an RSS that gives you the most recent released shows. Not specific to any type. So you shouldn't have a &q= in the url, or it should be empty. So just search for English translated shows, nothing in the search field. Then get that RSS for the first field in taiga.  

Taiga just gets a list of shows then, and you can chose what I picks for you with the filter options in the settings. 

The amount of results depends on the site, I know another site that I don't remember the name off  you could specify the amount of results in the url. But the first page should do for recent shows when it checks daily or every few hours. For searches you do, it's either best to be specific, you usually find what you need, or you can go to other sites manually if there's nothing good. Specifically older shows can be a bit hard to find. 

In short, first field only gets a type of shows, like English translated. Taiga filters it for you how you want with filter options. (In the settings)

**Comment by taigalover** [comment]
and is there a way to tell it what type of episode i want like for example, i want it to download one piece 1080 from anime-rg less than 300 mb and in its normal "one piece" folder (which already exists) every week when its airs

**Comment by Thomasedv** [comment]
Read my last comment, made an edit just now.

There is no way to download to folders for specific shows as far as I know. 

Selecting 1080p and sub group is done in filter settings. I urge you to explore and try it out and see what taiga highlights, they are a bit hard to explain and depends a bit on what you want. I think you can pick based on filesize, but I haven't tried it and don't remember if it was an option.

**Comment by taigalover** [comment]
alrite, i took out the %title% for the first rss space and added %title% to the second so now when i click the "check new torrents button" its different from the first page of nyaa.si english translated page, its mixed in some giberish and japanese, maybe i didnt pick up the right rss, or is it supposed to be like that?

im sorry to ask this of you but can you please help me get a working rss that is for english translated.

im reeally sorry again
